astrocloud release 1.4.0 (#602)
astronomer/docs: astro/cli-release-notes.md

If you have any questions or a bug to report, don't hesitate to reach out to us via Slack or Intercom. We're here to help.

## v1.4.0

Release date: April 14, 2022

> ### New Command to Create Environment Variables

Reviewer: ```suggestion
### New Command to Create and Update Environment Variables
```

Users can now list Deployment Environment Variables and save them to a local env file with astrocloud deployment variable list. With this command users no longer have to copy and paste deployment variables from the Astronomer UI to use a deployments variables locally

> Users can also list a Deployment’s variables at any time. If a variable’s value is secret only the variable’s key will show in the output and be exported to an env file.

Reviewer 2: ```suggestion
You can now list a Deployment's Environment Variables and save them to a local `.env` file with the new `astrocloud deployment variable list` command, which makes it easy to export environment-level settings from a Deployment and test them in a local Airflow environment. 
```

> `astrocloud deployment variable create` is a new Astro CLI command that allows you to create and update [environment variables](environment-variables.md) for a Deployment on Astro. New environment variables can be loaded from a file (e.g. `.env`) or specified as inputs to the CLI command itself. If you already set environment variables [via a `.env` file locally](develop-project.md#set-environment-variables-via-env-local-development-only), this command allows you to set environment variables on Astro from that file as well. More generally, this command makes it easy to automate creating or modifying environment variables instead of setting them manually via the Cloud UI.

Reviewer: ```suggestion
`astrocloud deployment variable create` is a new command that allows you to create and update [environment variables](environment-variables.md) for a Deployment via the Astro CLI. New environment variables can be loaded from a file (e.g. `.env`) or specified as inputs to the CLI command itself. If you already set environment variables [via a `.env` file locally](develop-project.md#set-environment-variables-via-env-local-development-only), this command allows you to set environment variables on Astro from that file as well. More generally, this command makes it easy to automate creating or modifying environment variables instead of setting them manually via the Cloud UI.
```
